.circleci/config.yml

```diff
@@ -1,71 +1,79 @@
 defaults: &defaults
-  working_directory: ~/CLARK-Gateway
+  working_directory: ~/clark-gateway
   docker:
     - image: cyber4all/circleci-aws:node-alpine
 orbs:
-  aws-cli: circleci/aws-cli@1.2.1
+  node: circleci/node@3.0.0
+  docker: circleci/docker@1.4.0
 commands:
-  install-aws: aws-cli/install
-  setup-aws-cli: aws-cli/setup
+  install-docker: docker/install-docker
 version: 2.1
 jobs:
-  build:
+  build: 
     <<: *defaults
     steps:
       - checkout
+      - install-docker
+      - setup_remote_docker
       - run:
-          name: Install Dependencies
-          command: npm install
+          name: Install dependencies
+          command: |
+            npm install
       - run:
-          name: Build Application
-          command: npm run build
-      - persist_to_workspace: 
-          root: ./
-          paths: 
-              - ./dist
-              - ./package.json
-              - ./package-lock.json
-              - ./.elasticbeanstalk
-              - ./docs
-              - ./Procfile
-  deploy:
+          name: Build Image
+          command: | 
+            docker build -t clark-gateway:$CIRCLE_SHA1 -f Dockerfile.prod .
+      - run: 
+          name: Login to docker
+          command: docker login -u ${DOCKER_USERNAME} -p "${DOCKER_PASSWORD}"
+      - run: 
+          name: Publish Image
+          command: | 
+            docker tag clark-gateway:$CIRCLE_SHA1 "${DOCKER_USER_ORG}/clark-gateway:${CIRCLE_SHA1}"
+            docker push "${DOCKER_USER_ORG}/clark-gateway:${CIRCLE_SHA1}"
+  deploy-production:
     <<: *defaults
     steps:
-      - attach_workspace : 
-          at: ~/CLARK-Gateway
-      - setup-aws-cli:
-          aws-access-key-id: AWS_EB_AccessKey
-          aws-region: AWS_REGION_N_VA
-          aws-secret-access-key: AWS_EB_SecretKey
-          version: "1"
-      - run:
-          name: Install prod dependencies
-          command: npm install --only=prod
+      - attach_workspace:
+          at: ~/clark-gateway
+      - checkout
+      - install-docker
+      - setup_remote_docker
       - run: 
-          name: Zip dist
```

Comment:
This is a little random but would it be worth going through the versions already deployed in elasticbeanstalk and removing the old dist folders? I imagine that they're taking up a lot of room on S3.

Reply:
We can go back and update it.  The older dist folders are only like 11kb but the newer ones are 5mb.  So it may be worth it just to save on some space.